src/libraries/System.Configuration.ConfigurationManager/ref/System.Configuration.ConfigurationManager.cs

```diff
@@ -1052,6 +1052,7 @@ public partial class SettingsAttributeDictionary : System.Collections.Hashtable
     {
         public SettingsAttributeDictionary() { }
         public SettingsAttributeDictionary(System.Configuration.SettingsAttributeDictionary attributes) { }
+        protected SettingsAttributeDictionary(System.Runtime.Serialization.SerializationInfo serializationInfo, System.Runtime.Serialization.StreamingContext streamingContext) { }
```

Comment:
I assume there wasn't a reason why we were explicitly hiding these from the ref file before, right?

Reply:
This was an omission and not caught by reverse api compat for some reason. There wasn't a reason for it to be missing, right.